crates/indexer/src/geyser/programs/candy_machine.rs

```diff
@@ -35,12 +35,12 @@ pub fn parse_cm_config_lines(
     // This could occur if this function is called on a candy machine that uses hiddensettings instead of
     // config lines
     let bytes_needed_for_taken_bitmask = items_available / 8 + (items_available % 8).min(1);
-    let expected_taken_len = taken_bitmask_start + bytes_needed_for_taken_bitmask;
+    let total_expected_data_len = taken_bitmask_start + bytes_needed_for_taken_bitmask;
```

Comment:
renamed this for clarity since "taken" in this context refers to the length of the "taken" bitmask